setup.py

```diff
@@ -61,6 +56,23 @@ def run_tests(self):
         errno = pytest.main(self.pytest_args)
         sys.exit(errno)
 
+
+test_path = os.path.join(root_path, 'odl/test')
+
+
+def find_tests():
+    tests = []
+    for path, _, filenames in os.walk(os.path.join(root_path, test_path)):
+        for filename in filenames:
+            basename, suffix = os.path.splitext(filename)
+            if (suffix == '.py' and
+                    (basename.startswith('test') or
```

Comment:
It is actually supposed to be `_test` and `test_`

See:

http://doc.pytest.org/en/latest/goodpractices.html